pkg/preflight/preflight.go

```diff
@@ -216,6 +229,21 @@ func Run(appID string, appSlug string, sequence int64, isAirgap bool, archiveDir
 	return nil
 }
 
+func preflightState(p troubleshootpreflight.UploadPreflightResult) string {
```

Comment:
why not use the existing `GetPreflightState` function`?

Reply:
The state from `GetPreflightState` is of the entire list of preflights i.e if there is an error in any, the state == `error`. `preflightState` gets the state of each preflight for logging purposes.